pom.xml

```diff
@@ -75,6 +75,24 @@
         <bootclasspath.compile>${bootclasspath.java6}</bootclasspath.compile>
         <bootclasspath.testCompile>${bootclasspath.java6}</bootclasspath.testCompile>
         <antlr.testinprocess>true</antlr.testinprocess>
+        <maven-bundle-plugin.version>2.3.7</maven-bundle-plugin.version>
+
+        <!-- Default OSGi metadata - see maven-bundle-plugin configuration -->
+        <osgi.symbolicName>${project.groupId}.${project.artifactId}</osgi.symbolicName>
+        <osgi.failok>false</osgi.failok>
+        <osgi.nouses>false</osgi.nouses>
+        <osgi.versionPolicy>${range;[===,=+);${@}}</osgi.versionPolicy>
```

Comment:
Hi Sam, first off I apologise that it has taken this long to reply, I thought this was dead and wasn't paying attention to any update until Terence emailed me directly today. In the version policy the '@' macro means that the version of a package as it is provided in a Maven defined dependency in the build classpath, this is an argument to the range macro. BND macro syntax is a little cryptic. I believe that the use of the '@' here may be redundant but that is the way I have been using it and it has the desired effect.